server/meshmodel/kubernetes/v1.25.2/v1.0.0/relationships/edge-network-statefulset-service.json

```diff
@@ -0,0 +1,87 @@
+{
+    "schemaVersion": "relationships.meshery.io/v1alpha2",
```

Comment:
@Jougan-0 given that we're mid-flight in migrating to v1alpha3, it seems like we'll want to encourage @TomiwaAribisala-git to use the v1alpha3 format. Do we have an example or two readily handy that @TomiwaAribisala-git can be pointed to?

Reply:
Alright, Noted